common/control_files/vrouter-dpdk-post-start.sh

```diff
@@ -0,0 +1,71 @@
+#! /bin/bash
+##
+## The script is based on tools/packaging/common/control_files/vrouter-pre-start.sh
```

Comment:
This file is actually a copy of vrouter-pre-start.sh. Only line that differ is that it calls vrouter_dpdk_start not insert_vrouter like in the if-vhost0-dpdk file above. I don't think this change will be accepted by the Juniper packaging repo maintainers as we copied the same function. Each time someone change for example create_virtual_gateway(), it has to remember to change it in two files. In general duplication of the same code is not good idea.